modules/ROOT/pages/index.adoc

```diff
@@ -12,81 +15,203 @@ Operations and development teams use the monitoring tools to diagnose issues and
+//feature tier:
+include::reuse::partial$billing/pricing.adoc[tag=featureAdvReqMonitoring]
 
-== Raw Data Downloads
+Download warehoused (archived) log data for a Mule app, for example, to help investigate security, compliance, or other issues that logs can reveal.
 
-You can download warehoused log data on a Mule app, for example, to help investigate  security, compliance, or other issues that logs can reveal.
+See xref:raw-data.adoc[].
 
-See xref:raw-data.adoc[Raw Data].
+[[tools]]
+=== Tools for Monitoring Integrations
 
-== Tools for Monitoring Integrations
+//feature tier:
+include::reuse::partial$billing/pricing.adoc[tag=featureAdvReqMonitoring]
 
-You can link to connectors that help deliver Anypoint Monitoring notifications to external systems, including custom metrics connectors.
+Use connectors that help deliver Anypoint Monitoring notifications to external systems, including custom metrics connectors.
 
-See xref:tools.adoc[Tools].
+See xref:tools.adoc[].
 
-== API Functional Monitoring
+[[functional-monitoring]]
+=== API Functional Monitoring
 
 Monitor APIs and endpoints at scheduled intervals. Receive reports about whether tests pass or fail, and view statistics about API and endpoint performance.
 
-See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
+See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring]. 
+
+//Rajesh 11/24: retain data-storage-retention
+[[data-storage-retention]]
+=== Data Storage and Retention
+
+Data storage and retention limits are determined by your pricing plan.
+include::reuse::partial$billing/pricing.adoc[tag=featureAdvLimits]
+
+include::reuse::partial$billing/pricing.adoc[tag=dataRetentionMonitoring]
+
+See <<billing>>. For complete details about pricing, contact your customer support representative. 
 
-== Anypoint Monitoring Packages and Plans
+[[billing]]
+== Anypoint Platform Packages and Plans
 
-Anypoint Monitoring packages and plans are available:
+Feature availability in Anypoint Monitoring depends on the tier of your Anypoint Platform pricing package or plan:
 
 * <<plans-usage-based>>
 * <<plans-legacy>>
 
 [[plans-usage-based]]
 === Anypoint Monitoring Features for Usage-Based Packages
 
-Anypoint Monitoring is the standard method of monitoring Mule application and API performance so that you can more quickly identify and resolve issues. All organizations receive fundamental monitoring regardless of pricing model. However, organizations that have the Anypoint Integration Advanced package receive greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
+Anypoint Monitoring is the standard method of monitoring Mule app and API performance so that you can more quickly identify and resolve issues. 
```

Comment:
make this a partial?

Reply:
Adding this suggestion to[W-17361464](https://gus.lightning.force.com/a07EE000026JsovYAC)